packages/jest-cli/src/TestRunner.js

```diff
@@ -67,18 +75,40 @@ class TestRunner {
     return this._buildPromise;
   }
 
-  _getAllTestPaths() {
-    return this._hasteMap
-      .matchFiles(this._testRegex)
-      .then(paths => paths.filter(path => this.isTestFilePath(path)));
+  _filterTestPathsWithStats(allPaths, options) {
+    const data = {
+      total: allPaths.length,
```

Comment:
can you add `paths` here too? I don't like changing the structure of objects later on.